src/Domain/SalesforceWriteProxyRepository.php

```diff
@@ -14,6 +14,9 @@
  */
 abstract class SalesforceWriteProxyRepository extends SalesforceProxyRepository
 {
+    /** Maximum of each type of pending object to process */
+    private const int MAX_PER_BULK_PUSH = 400;
```

Comment:
phpcs doesn't know that consts can have types. So far I've just avoided putting types on consts. If they're private then the type is fully inferable anyway.

Reply:
Thanks, reached the same conclusion